libs/designsystem/src/lib/components/list/list.component.ts

```diff
@@ -159,13 +171,98 @@ export class ListComponent implements OnInit, AfterViewInit, OnChanges {
     }
   }
 
+  private _headerFn(item: any, index: number, items: any[]) {
+    return this.headerTemplate && index === 0 ? true : null;
+  }
+
+  headerFn = this._headerFn.bind(this);
+
+  private _sectionHeaderFn = (item: any, index: number, items: any[]) => {
+    return this.sectionNameMap.get(index);
+  };
+
+  sectionHeaderFn: (
+    item: any,
+    index: number,
+    items: any[]
+  ) => string | null = this._sectionHeaderFn.bind(this);
+
+  private _footerFn(item: any, index: number, items: any[]) {
+    return this.footerTemplate && items && items.length > 0 && items.length - 1 === index
+      ? true
+      : null;
+  }
+
+  footerFn = this._footerFn.bind(this);
+
+  private kirbyItemHeightFn(item: any, index: number) {
+    const dividerHeight = 1;
+    if (this.isFirst(item) || this.isLast(item)) {
```

Comment:
Replace with 1 call to an orderMap that returns both isFirst+isLast as in:
https://github.com/kirbydesign/designsystem/blob/6d05e2a6d1c6f193c4f44fcbafe6d6cce5ce9de0/src/kirby/components/list/list.component.ts#L221-L234